testing/integration/linux_deb_test.go

```diff
@@ -49,6 +50,75 @@ func TestDebLogIngestFleetManaged(t *testing.T) {
 	agentFixture, err := define.NewFixtureFromLocalBuild(t, define.Version(), atesting.WithPackageFormat("deb"))
 	require.NoError(t, err)
 
+	installOpts := atesting.InstallOpts{
+		NonInteractive: true,
+		Force:          true,
+		InstallServers: false,
+	}
+
+	testDebLogIngestFleetManagedWithCheck(ctx, t, agentFixture, info, installOpts,
```

Comment:
This check should go at the start and end of the upgrade test as well, perhaps just the end since we know from this test that installation behaves properly.

The same should apply to the RPM upgrade variant.

Reply:
followup